Assets/Scripts/Game/UserInterfaceWindows/DaggerfallPotionMakerWindow.cs

```diff
@@ -122,11 +124,6 @@ public override void OnPush()
             Refresh();
         }
 
-        public override void OnPop()
-        {
-            ClearCauldron();
-        }
-
```

Comment:
I think OnPop() should still be here and it should clear the ingredients item list and allow the cloned items to be disposed of. Just do:

`ingredients = null;`

What do you think? (I'm theory crafting here)

Reply:
Ah yes, because the window is reused instead of destroyed